meeting_minutes/220224_SARIF_TC_53.md

```diff
@@ -0,0 +1,175 @@
+  * Code Insights Protocol (CIP) might be a focus point for defining the extent of metrics scope
+* Stefan: As long as we end up with domains that are numbers or categories i.e. leading to aggregates and trends and no free text
+
+## 3.5 Review proposal for what's in- and out-of-scope - general discussion
+
+None - but keep item for future discussions
+
+## 3.6 Status of Wikipedia page
+
+* David: Has been granted a Wikipedia account and can continue to upload.
+
+## 3.7 Discuss end-to-end results management (including code insights protocol)
+
+None
+
+# 4. Other Business
+
+* Michael: 
+  * Suggests considering setting some specific gaols for the next steps of standard evolution
+    * Initial simple candidate sources for dynamic analysis (scenarios):
+      * Crash reports
+      * Fuzzing
+      * Telemetry
+    * More scary sources
+      * Web endpoint scanning
+      * Accessibility
+  * Example: Next two weeks try to tackle the above steps in order?
+* Nathan:
+  * Works with crash reports - typically more than a single stack how a specific crash or "class" of crash can occur
+  * Many internal and external tools produce results in very similar form for the crash reports (related to CIP)
+  * All stored in static location similar to typical SARIF (local) storage model
+  * Interested in seeing how a remote storage approach would look like - alignment with source repository storage
+  * Explains the current approach to telemetry exploration within his team
+    * Broad brainstorming like approach and
+    * Specifically guiding developers e.g. hinting at 
+      * not only creating telemetry, but also 
+      * monitoring and using the data (observability) 
+* Michael: Plans to provide results from his, Nathan's, Eddy's team and others in two weeks time
+* All discuss crash report and telemetry integrations
+* All discuss how information of several crash reports is typically projected into buckets for guiding developers and that this is strongly tool dependant
+* Michael: Reminds on how SARIF extended the use cases as interchange format across boundaries (between tools)
+* Stefan: 
+  * When we have the first content ideas / models for crash and telemetry data inclusion, then 
+  * we should evaluate the following aspects:
+    * composing (how can the diverse tools access elements for projecting and aggregating)
+    * writing (latency and throughput)
+    * remote storing and retrieving (discovery and addressing)
+    * reading (latency and throughput)
+
+# 5. Resolutions and Decisions reached (by 10 minutes prior to scheduled meeting end)
+
+## 5.1 End debate of other issues by 10 minutes prior to scheduled meeting end and follow the agenda from this point (Co-Chair Keaton)
+
+## 5.2 Review of Decisions Reached (Secretary Hagen)
+
+None
+
+## 5.3 Review of Action Items (Secretary Hagen)
+
+* ACTION on David: To please include an agenda item on the reporting of the two week evakuation effort
```

Comment:
evakuation?

Reply:
@michaelcfanning please kindly verify that "evaluation" was the term meant ... thanks.